packages/web-components/fast-foundation/custom-elements-manifest-plugins.mjs

```diff
@@ -0,0 +1,141 @@
+ * @param  {Partial<import("custom-elements-manifest/schema").Module>} moduleDoc
+ * @param  {string} name
+ * @return {import("custom-elements-manifest/schema").Declaration}
+ */
+function getDeclarationDoc(moduleDoc, name) {
+    return moduleDoc.declarations.find(x => x.name === name);
+}
+
+/**
+ * Fix a Node's JSDoc link comments
+ * @param {import('typescript').Node & { jsDoc: import('typescript').JSDoc[] }} node
+ * @param {import("custom-elements-manifest/schema").Declaration} doc
+ * @param {*} context
+ * @param {number} [offset=0]
+ */
+function fixLinks(node, doc, context, offset = 0) {
+    node?.jsDoc?.forEach(jsDoc => {
+        const parsed = parse(jsDoc?.getFullText());
+        parsed?.forEach(parsedJsDoc => {
+            if (
+                parsedJsDoc.description &&
+                parsedJsDoc.description.indexOf("{@link") >= 0
+            ) {
+                doc.description = parsedJsDoc.description.trim();
+            }
+        });
+    });
+}
+
+/**
+ * @return {import('@custom-elements-manifest/analyzer').Plugin}
+ */
+export function jsdocLinkFix() {
+    return {
+        name: "jsdoc-example",
+        analyzePhase({ ts, node, moduleDoc, context }) {
+            if (!hasJsDocComments(node)) return;
+            if (ts.isClassDeclaration(node)) {
+                const className = node.name.getText();
+                const classDoc = getDeclarationDoc(moduleDoc, className);
+                if (!classDoc)
+                    return (
+                        context.dev &&
+                        console.warn(
+                            `[jsdoc-example]: Could not find class ${className} in module doc for path ${moduleDoc.path}`
+                        )
+                    );
+                fixLinks(node, classDoc, context);
+            } else if (ts.isPropertyDeclaration(node)) {
+                const propertyName = node.name.getText();
+                const className = node.parent?.name?.getText?.();
+                const classDoc = getDeclarationDoc(moduleDoc, className);
+                if (!classDoc) return;
+                const isStatic = node.modifiers?.some?.(
+                    x => x.kind === ts.SyntaxKind.StaticKeyword
+                );
+                const propertyDoc = getClassMemberDoc(
+                    moduleDoc,
+                    className,
+                    propertyName,
+                    isStatic
+                );
+                if (!propertyDoc)
+                    return (
+                        context.dev &&
+                        console.warn(
+                            `[jsdoc-example]: Could not find property ${propertyName} of ${className} in module doc for path ${moduleDoc.path}`
+                        )
+                    );
+                fixLinks(node, propertyDoc, context, 1);
+            } else if (ts.isMethodDeclaration(node)) {
+                // TODO: handle object methods
```

Comment:
Is this todo still open? Should it have a tracking issue?

Reply:
Ah, no. Was copied over from the example plug-in code I used. Removed.